Services/Concurrency/RateLimiting.cs

```diff
@@ -58,6 +60,50 @@ public class RateLimiting : IRateLimiting
             log.Info("Rate limiting started. This message should appear only once in the logs.");
         }
 
+        public Models.Simulation.SimulationRateLimits GetCounters()
```

Comment:
could you change the method name? the method doesn't return counters, does it?

Reply:
done